scripts/deploy.sh

```diff
@@ -51,6 +52,11 @@ ansible-galaxy install --role-file=/opt/rpc-openstack/ansible-role-requirements.
 # Enable playbook callbacks from OSA to display playbook statistics
 grep -q callback_plugins playbooks/ansible.cfg || sed -i '/\[defaults\]/a callback_plugins = plugins/callbacks' playbooks/ansible.cfg
 
+#Clone Magnum repository if required
+if [[ "$DEPLOY_MAGNUM" == "yes" ]]; then
+  git clone https://github.com/openstack/openstack-ansible-os_magnum.git -b stable/newton --single-branch $OA_DIR/playbooks/roles/os_magnum
```

Comment:
It's likely better to submodule the role repo to ensure that the SHA is properly fixed and managed.